src/driver/sqlite-abstract/AbstractSqliteQueryRunner.ts

```diff
@@ -1124,15 +1148,16 @@ export abstract class AbstractSqliteQueryRunner extends BaseQueryRunner implemen
      */
     protected createIndexSql(table: Table, index: TableIndex): Query {
         const columns = index.columnNames.map(columnName => `"${columnName}"`).join(", ");
-        return new Query(`CREATE ${index.isUnique ? "UNIQUE " : ""}INDEX "${index.name}" ON "${table.name}" (${columns}) ${index.where ? "WHERE " + index.where : ""}`);
+        const [database, tableName] = this.splitTablePath(table.name);
+        return new Query(`CREATE ${index.isUnique ? "UNIQUE " : ""}INDEX IF NOT EXISTS ${database ? `"${database}".` : ""}${this.escapePath(index.name!)} ON "${tableName}" (${columns}) ${index.where ? "WHERE " + index.where : ""}`);
```

Comment:
`IF NOT EXISTS` is something new here. Did you have particular problems without it?

Reply:
Honestly, I can't remember. I have a feeling it was required though. 

Anyways, i have removed it and will re-submit if my testing shows a need